+neuron/Object.m

```diff
@@ -72,11 +72,10 @@ function delete(self)
         % Decrease refcount by 1; if refcount is 0, release the C++ Object.
         %   delete()
 
-            self.obj.refcount = self.obj.refcount - 1;
+            clib.neuron.hoc_obj_unref(self.obj);
 
             % We need to do this, or else we get crashes.
             if self.obj.refcount == 0
```

Comment:
You've tested to make sure that this works? I'd worry that if refcount goes to 0 and NEURON releases the memory, that we wouldn't be able to check self.obj.refcount... unless the need for `clibRelease` is somehow keeping us safe?

Reply:
Yes, I tested it - checking `refcount` worked.

With this version it even works without `clibRelease`, so checking `refcount` is not even necessary. So that issue has been solved then. The `n.List()` counts are 0, and the tests run without issue.